route-rs-runtime/src/composite/mton_composite.rs

```diff
@@ -0,0 +1,157 @@
+use crate::link::{Link, LinkBuilder, PacketStream, JoinLinkBuilder, TeeLinkBuilder};
+
+pub struct  MtoNCompositeBuilder<Packet: Sized + Send + Clone> {
+    in_streams: Option<Vec<PacketStream<Packet>>>,
+    join_queue_capacity: usize,
+    tee_queue_capacity: usize,
+    branches: Option<usize>,
```

Comment:
Is there a better name than `branches` to express the number of egressors the `Link` will produce? How about `num_egressors`?

Reply:
ack, good suggestion.

Reply:
Done